### PR TITLE
python3Packages.pandas: 1.4.4 -> 1.5.2

### PR DESCRIPTION
**Pull request by nomeata**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by nomeata** [comment]
Yuck, the build is unreliable (https://github.com/pandas-dev/pandas/issues/47305)

File: pkgs/development/python-modules/pandas/default.nix [hunk 1]
```diff
@@ -96,6 +96,8 @@ buildPythonPackage rec {
     # than expected, e.g. on amd64 with FMA or on arm64
     # https://github.com/pandas-dev/pandas/issues/38921
     "test_rolling_var_numerical_issues"
+    # Require mathplotlib
```
Comment (risicle) [inline pkgs/development/python-modules/pandas/default.nix:99]:
```suggestion
    # requires matplotlib
```

Comment (risicle) [reply pkgs/development/python-modules/pandas/default.nix:99]: Also please squash your fixup commit

**Comment by risicle** [comment]
Also, this will need to be retargeted at staging using this technique due to the large number of rebuilds: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging

**Comment by risicle** [comment]
@ofborg build python3Packages.fastparquet

Fails for me, aarch64-linux

**Comment by nomeata** [comment]
Sorry for doing the rebase in the codeowners-annoying way. I’ll be more careful next time.

**Comment by Mic92** [comment]
@ofborg build python3.pkgs.pandas

**Comment by dotlambda** [comment]
@ofborg eval

**Comment by dotlambda** [comment]
Please update to 1.5.2.

**Comment by nomeata** [comment]
@ofborg build python3.pkgs.pandas

**Comment by 7c6f434c** [comment]
@ofborg build python3Packages.fastparquet

**Comment by dotlambda** [comment]
@ofborg build python3.pkgs.pandas python3Packages.fastparquet

**Comment by mweinelt** [comment]
Cherry-picked into python-updates. #208232

**Pull request closed without merging**